app/src/pages/instance/organizationsPage/organizationsPageHeader/organizationsPageHeader.scss

```diff
@@ -50,25 +50,24 @@
     align-items: center;
     justify-content: center;
 
+    svg {
```

Comment:
Looks like this can affect all the nested SVGs that can be inside the `.icons`.
Let's avoid modifying styles for indirect children where possible.
That bug is also valid for `OrganizationProjectsPage`.

I suggest to review:
1. The search component in the ui-kit and replace it with actual search icon from [DS icons](https://www.figma.com/design/gjYQPbeyf4YsH3wZiVKoaj/%F0%9F%9B%A0-RP-DS-6?node-id=136-2&node-type=frame&t=S4cEZ8lwY2w0djGv-0).
2. `BaseIconButton` component usage inside the `SearchField` to automatically receive needed styles for all states described in DS.

Reply:
Done.